### DISCUSSION TITLE
Mode miroir

### DISCUSSION THREAD

**Discussion by blogmotion** [luminescence-software/Echosync-Support#4] [Ideas]
Bonjour,

Est-il possible d'implémenter un mode miroir ? de sorte à avoir une suppression des fichiers/dossiers dans la destination qui ne sont plus présents dans la source ?

J'aimerais remplacer le logiciel "[Create Synchronicity](https://synchronicity.sourceforge.net/screenshots.html)" par EchoSync mais il me manque cette option :)

Merci

**Comment by CyberSinh** [comment]
Bonjour,

Je ne suis pas sur de comprendre votre question.

En cochant les cinq [options de synchronisations](https://www.luminescence-software.org/fr/echosync/documentation#quel-type-de-difference-e), vous aurez bien un fonctionnement en "mode miroir".

**Reply by blogmotion** [reply]
Merci pour la réponse, effectivement en combinant les 5 options cela semble correspondre à un mode miroir, désolé je n'avais pas eu l'idée de cocher les 5. Est-ce qu'il serait possible d'ajouter une case "miroir" qui cocherait les 5 options d'un coup ?

**Reply by CyberSinh** [reply]
Cela alourdirait l'interface sans grande valeur ajoutée.
